### DISCUSSION TITLE
Quiz

### DISCUSSION THREAD

**Discussion by sebasvelasco353** [sebasvelasco353/FDP-2022_01#170] [Show and tell]
Describir cómo y para que usamos clases en JavaScript.

**Comment by Juli0931** [comment]
Las clases son una plantilla que utilizamos al momento de crear objetos, pueden ser públicas o privadas y requieren de un método constructor que se encarga de asignar ciertos parámetros iniciales. Para utilizarlos, declaramos la clases con la palabra "class" y luego asignamos los atributos con "this".

**Comment by Juanaa16** [comment]
la clase es una plantilla, o una etiqueta para representar una entidad u objeto, como lo puede ser un usuario, contenido, imagen o alerta, sirve para representar cualquier sustantivo, permitendo realizar funciones y características especificas a cada objeto

**Comment by Santygr21** [comment]
Las clases las usamos para agrupar un conjunto de atributos y métodos que una cantidad x de objetos van a tener en común y así poder crear objetos de manera más sencilla, ya en JavaScript cómo lo usamos es declarando las clases con la palabra "class", luego dándole los atributos con this.

**Comment by AlejandroRv19** [comment]
Para hacer una clase se utiliza la palabra  class y se le da un nombre a la clase y luego entre llaves se declaran el nombre de la clase.
las clases se usan para mostrar objetos con sus atributos y metodos.

**Comment by sophienunez** [comment]
las clases se usan para almacenar en ellas todo lo que tenga relación con un ''tema'' en específico. Ejemplo de declaración de una clase:
class Animal {}
de esta clase se pueden derivar objetos así: const pato = new Animal();

**Comment by Darksea1220** [comment]
Para usar clases en JS empezamos escribiendo class, luego el nombre del objeto, llaves, un constructor con las características del objeto y luesgo las características con this, quedaría algo así: 
`class objeto{
constructor (tamaño, color){
tihs.tamaño=tamaño
this.color=color}
}`
y sirven para dar una sintaxis mucho más clara y simple para crear objetos y lidiar con la herencia.

**Comment by AnnaHenao131** [comment]
Las clases sirven para la creación de objetos de datos según un modelo predefinido. Las clases se utilizan para representar entidades o conceptos como los sustantivos en el lenguaje.
Cada clase es un modelo que define un conjunto de variables

**Comment by DilanStivenCano** [comment]
Una clase es una manera de organizar código de forma entendible con el objetivo de simplificar el funcionamiento de nuestro programa, y las podemos usar para mostrar objetos

**Comment by Kathereyes19** [comment]
Una clase la utilizamos para organizar  código de forma entendible agrupando atributos y métodos con el objetivo de simplificar el funcionamiento de nuestro programa. Además las clases son concepto abstractos de los que se pueden crear objetos de programación, cada uno con sus características concretas. Las clases se declaran con la palabra clave class.

**Comment by DParuma** [comment]
Las clases en javascript nos permiten de forma corta y rápida llamar características o parametros que posea un objeto que volveremos a usar mediante un constructor ademas podemos heredar o modificar el mismo cuando lo llamemos.

**Comment by Axelito02** [comment]
Las clases en los objetos sirven para definir la estructura simple ytambién sirve para acceder a características específicas del objeto o dicho de otra manera heredar características del objeto.

**Comment by IBaronap** [comment]
Las clases en JavaScript dan una sintaxis más clara y simple para crear objetos y lidiar con la herencia. Para definir una se hace una declaración de clase, usando la palabra reservada class y un nombre para la clase (Ej. class Rectangulo {} ).

**Comment by salazar2356** [comment]
Ahí van los metodos y atributos que van a tener los objetos que se creen.

**Comment by ghost** [comment]
> Describir cómo y para que usamos clases en JavaScript.

Las clases de JavaScript proveen una sintaxis mucho más clara y simple para crear objetos y lidiar con la herencia.

**Comment by Karolfc** [comment]
Las clases en JS se usan para para dar una sintaxis más clara y simple con el fin de lidiar con la herencia y crear nuevos objetos. Las clases son tipos de funciones, por ello, se declaran con la palabra "class {}".

**Comment by Marianaao** [comment]
Las clases en JS es un tipo de función que nos permite de una manera mas sencilla lograr que las caracteristicas del codigo que estemos programando se ordene para que sea mas simple batallar con la creación de objetos y la herencia. Se define como ejemplo "class circulo { ... }"
